<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddABQuestDialog</class>
 <widget class="QDialog" name="AddABQuestDialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>582</width>
    <height>288</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AddABQuestDialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="2" column="2">
    <widget class="QLineEdit" name="rateLine"/>
   </item>
   <item row="2" column="3">
    <widget class="QPushButton" name="okButton">
     <property name="text">
      <string>ОК</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Баллы:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QPushButton" name="cancelButton">
     <property name="text">
      <string>Отмена</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" colspan="5">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Вопрос:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QRadioButton" name="bRadio">
       <property name="text">
        <string>b</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QRadioButton" name="dRadio">
       <property name="text">
        <string>d</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QRadioButton" name="cRadio">
       <property name="text">
        <string>c</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QRadioButton" name="aRadio">
       <property name="text">
        <string>a</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="2" colspan="2">
      <widget class="QLineEdit" name="aLine">
       <property name="enabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="2" colspan="2">
      <widget class="QLineEdit" name="bLine"/>
     </item>
     <item row="4" column="2" colspan="2">
      <widget class="QLineEdit" name="cLine"/>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Правильный ответ:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2" colspan="2">
      <widget class="QLineEdit" name="dLine"/>
     </item>
     <item row="1" column="2" colspan="2">
      <widget class="QLineEdit" name="RALine"/>
     </item>
     <item row="0" column="2">
      <widget class="QPlainTextEdit" name="questLine"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
